tests/python/gaia-ui-tests/gaiatest/apps/system/regions/fxaccounts.py

```diff
@@ -17,6 +17,15 @@ class FirefoxAccount(Base):
     _password_error_locator = (By.ID, 'fxa-error-overlay')
     _password_error_close_locator = (By.ID, 'fxa-error-ok')
 
+    def __init__(self,marionette):
+        Base.__init__(self,marionette)
+
+        self._switch_to_fxa_iframe()
+        input_field = Wait(self.marionette, timeout=60).until(
+            expected.element_present(*self._email_field_locator))
+        Wait(self.marionette).until(expected.element_displayed(input_field))
+        Wait(self.marionette).until(lambda m: input_field.rect['x'] == 25)
```

Comment:
apparently this is the layout of the this element.  I could try locating the page view though, thought checking on the actual element would be more effective.